NextcloudTalk/CallViewController.m

```diff
@@ -926,10 +925,25 @@ - (void)adjustMoreButtonMenu
             speakerActionTitle = NSLocalizedString(@"Enable speaker", nil);
         }
 
-        UIAction *speakerAction = [UIAction actionWithTitle:speakerActionTitle image:speakerImage identifier:nil handler:^(UIAction *action) {
+        BOOL shouldShowAirPlayButton = audioController.numberOfAvailableInputs > 1;
+        if (shouldShowAirPlayButton) {
+            speakerImage = [UIImage systemImageNamed:@"airplayaudio"];
+            speakerActionTitle = NSLocalizedString(@"Audio options", nil);
+        }
+
+        void (^speakerBlock)(UIAction *action) = ^void(UIAction *action) {
             [weakSelf speakerButtonPressed:nil];
-        }];
+        };
+
+        void (^airplayBlock)(UIAction *action) = ^void(UIAction *action) {
+            for (id subview in self->_airplayView.subviews) {
```

Comment:
Pretty sure this will lead to a retain cycle again because we capture a strong reference to self. We need to do something like this:
https://github.com/nextcloud/talk-ios/blob/43cf853b6fe1b1e2aab08d28f7c693247410ecd1/NextcloudTalk/NCPeerConnection.m#L491-L494

Reply:
Good catch! :)